src/libraries/System.Net.Sockets/tests/FunctionalTests/UnixDomainSocketTest.cs

```diff
@@ -125,8 +124,8 @@ public async Task Socket_ConnectAsyncUnixDomainSocketEndPoint_NotServer()
         }
 
         [ConditionalFact(nameof(PlatformSupportsUnixDomainSockets))]
```

Comment:
speaking of my earlier comment, we can replace this with Socket.OSSupportsUnixDomainSockets since it is public now and get rid of the custom PlatformSupportsUnixDomainSockets code.

Reply:
I'm not opposed to this at all. I assumed there was some motivation behind the `PlatformSupportsUnixDomainSockets` bool (which helpfully just returns true for non-Windows)